internal/controller/consoleplugin_reconciler.go

```diff
@@ -60,11 +69,56 @@ func (r *ConsolePluginReconciler) Subscription() *controller.Subscription {
 				ObjectName:      TopologyConfigMapName,
 				EventType:       ptr.To(controller.DeleteEvent),
 			},
+			{
+				Kind:            ptr.To(ConfigMapGroupKind),
+				ObjectNamespace: r.namespace,
+				ObjectName:      ConsolePluginImagesConfigMapName,
+				EventType:       ptr.To(controller.CreateEvent),
+			},
+			{
+				Kind:            ptr.To(ConfigMapGroupKind),
+				ObjectNamespace: r.namespace,
+				ObjectName:      ConsolePluginImagesConfigMapName,
+				EventType:       ptr.To(controller.UpdateEvent),
+			},
+			{
+				Kind:            ptr.To(ConfigMapGroupKind),
+				ObjectNamespace: r.namespace,
+				ObjectName:      ConsolePluginImagesConfigMapName,
+				EventType:       ptr.To(controller.DeleteEvent),
+			},
```

Comment:
nit: you can remove `EventType` if you want to reconcile all event types, instead of needing to seperately match for each one
```suggestion
			{
				Kind:            ptr.To(ConfigMapGroupKind),
				ObjectNamespace: r.namespace,
				ObjectName:      ConsolePluginImagesConfigMapName,
			},
```